Add Conda install to install page (#5892)
nextflow-io/nextflow: docs/install.md

## Install Nextflow

Nextflow is distributed as an easy to use self-installing package. It is also distributed via Conda and as a standalone distribution.

> ### Self-installing

Reviewer: ```suggestion
### Self-install
```

    :::

    :::{warning}
    Nextflow updates its executable during the self-install process, therefore the update can fail if the executable is placed in a directory with restricted permissions.
    :::

4. Confirm Nextflow is installed correctly:

    ```{code-block} bash
    :class: copyable
    nextflow info
    ```

### Conda

To install Nextflow with Conda:

1. Add the `bioconda` and `conda-forge` channels to your Conda configuration:
    
    ```{code-block} bash
    :class: copyable
    conda config --add channels bioconda
    conda config --add channels conda-forge

>     ```

Reviewer 2: This is not strictly required. It should be enough specifying `bioconda::nextflow` when installing it

Author: I've removed this step and adjusted the other steps to account for specifying `bioconda::nextflow`. Thanks!

Reviewer 2: ok, thanks

2. Create an environment:

    ```{code-block} bash
    :class: copyable

>     conda create --name nf_env nextflow

Reviewer 2: ```suggestion
    conda create --name nf-env nextflow
```

Nit, it we favour kebab-case, over snake_case

>     Nextflow will update its executable during the self update process, therefore the update can fail if the executable is placed in a directory with restricted permissions.

Reviewer: ```suggestion
    Nextflow updates its executable during the self-install process, therefore the update can fail if the executable is placed in a directory with restricted permissions.
```

Reviewer: Wondering if it's meant to be self-update in the context of self-install?

    :::

4. Confirm Nextflow is installed correctly:

    ```{code-block} bash
    :class: copyable
    nextflow info
    ```

### Conda

To install Nextflow with Conda:

> 1. Add the Bioconda and conda-forge channels to your Conda configuration:

Reviewer: ```suggestion
1. Add the `bioconda` and `conda-forge` channels to your Conda configuration:
```

    
    ```{code-block} bash
    :class: copyable
    conda config --add channels bioconda
    conda config --add channels conda-forge
    ```

2. Create an environment:

    ```{code-block} bash
    :class: copyable
    conda create --name nf_env nextflow
    ```

> 3. Active the environment:

Reviewer: ```suggestion
3. Activate the environment:
```

    ```{code-block} bash
    :class: copyable
    source activate nf_env
    ```

4. Confirm Nextflow is installed correctly:

    ```{code-block} bash
    :class: copyable
    nextflow info
    ```

:::{warning}
Installing Nextflow via Conda may lead to outdated versions, dependency conflicts, and Java compatibility issues. Using the self-installing package is recommended for a more reliable and up-to-date installation.
:::

### Standalone distribution

> The Nextflow standalone distribution (i.e. the `dist` release) is a self-contained `nextflow` executable that can run without needing to download core dependencies at runtime. This distribution is useful for offline environments, as well as building and testing Nextflow locally.

Reviewer: ```suggestion
The Nextflow standalone distribution (i.e., the `dist` release) is a self-contained `nextflow` executable that can run without needing to download core dependencies at runtime. This distribution is useful for offline environments as well as building and testing Nextflow locally.
```

    ```{code-block} bash
    :class: copyable
    ./nextflow-24.10.1-dist run info
    ```

:::{note}
The standalone distribution will still download core and third-party plugins as needed at runtime.
:::

## Seqera Platform

You can launch workflows directly from [Seqera Platform](https://seqera.io/platform/) without installing Nextflow locally.

Launching from Seqera Platform provides you with:

- User-friendly launch interfaces.
- Automated cloud infrastructure creation.
- Organizational user management.
- Advanced analytics with resource optimization.

Seqera Cloud Basic is free for small teams. Researchers at qualifying academic institutions can apply for free access to Seqera Cloud Pro.

> See the [Seqera Platform documentation](https://docs.seqera.io/platform) for set-up information and tutorials to get started.

Reviewer: ```suggestion
See the [Seqera Platform documentation](https://docs.seqera.io/platform) for tutorials to get started.
```
